composer.json

```diff
@@ -16,6 +16,7 @@
   "homepage": "https://gorriecoe.github.io/silverstripe-link/en/",
   "require": {
     "silverstripe/framework": "^4",
+    "silverstripe/cms": "^4",
```

Comment:
Can you remove this please? The project works well on framework-only headless projects that don't require the CMS